lib/coophub/backends/backends.ex

```
@@ -158,6 +151,9 @@ defmodule Coophub.Backends do
 
       {:error, %HTTPoison.Error{reason: reason}} ->
         {:error, reason}
+
+      _ ->
+        {:error, nil}
```

Error swallowing here? Should return `{:error, reason}` to be logged from its caller.

---

good catch! that was added for test purposes in the development phase!